util/cron/test-gasnet-asan.bash

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+#
+# Test ASAN-compatible configuration with gasnet & ASAN on linux64, running
+# multilocale tests only.
+#
+
+CWD=$(cd $(dirname $0) ; pwd)
+source $CWD/common-gasnet.bash
+source $CWD/common-asan.bash
+source $CWD/common-localnode-paratest.bash
+
+export CHPL_NIGHTLY_TEST_CONFIG_NAME="gasnet-asan"
+
+export GASNET_QUIET=Y
```

Comment:
Makes me think putting this into common-gasnet, but I see bunch of other scripts use this. So this is also fine.